Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3
+
+RUN pip install pipenv
```

Comment:
I didn't mean that you remove all the spaces, just add one and not two together.
Also, you can group commands together if you think they should be together (related or kinda the same), and use one whitespace (`enter`) to separate some of them (but not two whitespaces).